### ISSUE TITLE
A few specific questions regarding your implementation

### ISSUE THREAD

**Issue by amdegroot** [balancap/SSD-Tensorflow#52]
Hey Paul, I have been trying to complete my port of SSD to PyTorch and have come across your repo multiple times along the way. 

Currently, without using all of the data augmentation, my model appears to be getting to a loss of around 0.05 at only 20k iterations on VOC07trainval (purely training without any validation along the way).  Did you ever run into this, or is this normal to your knowledge? (I haven't implemented mAP yet so that's next on my list).

Second, your example detections look incredible.  Is the car one on the README using COCO weights? When I test on that image I only have been able to pick up a few of the cars with VOC07 weights.  It could also be my NMS parameters I suppose.. 

Your work is extremely impressive, while you're further along than I am I can attest to the difficulty and time required to reimplementing this network :P!

**Comment by balancap** [comment]
Hi Max!
I have running into problems to while training on VOC07 or VOC12. I implemented a bit of data augmentation, but not as advanced as the original SSD Caffe, but I still get some training convergence problem. I was quite busy the last few weeks, and could not really investigate it further, but I now have a bit more free time and hope to be able to fix this issue.

I used the weights trained on COCO+VOC07+VOC12 for the example (and the default configuration of the notebook). I usually visualize the direct output of SSD network for debugging (with simple threshold at 0.5) : you get a good idea if the detection problem comes from the net or the NMS algorithm/parameters.

Thanks :) Your implementation is quite impressive too, I wish I knew more of PyTorch ! And for sure, it's quite a time-consuming project to reimplement this network. I exploded all my first estimates on how long it would take me!
